palace.pyx

```diff
@@ -1055,7 +1055,13 @@ cdef class Source:
 
     # TODO: set direct filter
     # TODO: set send filter
-    # TODO: set auxiliary send
+
+    def set_auxiliary_send(self, slot: AuxiliaryEffectSlot, send: int) -> None:
+        """Connect the effect slot to the given send path,
+        using the filter properties.
+        """
```

Comment:
Ah we totally forgot about making this a property.  Also the docstring is of `setAuxiliarySendFilter`.